lib/auxiliary/time_helper.rb

```diff
@@ -4,9 +4,11 @@
 module AutoHCK
   # Helper module
   module Helper
-    def time_to_seconds(time)
-      time.split(':').reverse.map.with_index { |a, i| a.to_i * (60**i) }
-          .reduce(:+)
+    def time_to_seconds(time_str)
+      days, time_part = time_str.split('.')
+      hours, minutes, seconds = time_part.split(':').map(&:to_i)
+      total_hours = (days.to_i * 24) + hours
+      (total_hours * 3600) + (minutes * 60) + seconds
```

Comment:
Why don't you use the original code for `time_part` parsing?

Reply:
i believe that code was more explicit and easier to understand at a glance